src/orange/orangeinp/IntersectRegion.cc

```diff
@@ -666,13 +666,10 @@ ExtrudedPolygon::ExtrudedPolygon(ExtrudedPolygon::VecReal2 const& polygon,
     x_range_ = this->calc_range(polygon, X);
     y_range_ = this->calc_range(polygon, Y);
 
-    // Store only non-collinear points
-    Real3 const extents{
-        x_range_[1] - x_range_[0], y_range_[1] - y_range_[0], 0};
-    real_type abs_tol = ::celeritas::detail::BumpCalculator(
-        Tolerance<>::from_default())(extents);
-
-    polygon_ = detail::filter_collinear_points(polygon, abs_tol);
+    // Store only non-collinear points. Use an absolute tolerance; otherwise,
+    // for example, an arbitrilarly large regular dodecagon becomes a hexagon
```

Comment:
```suggestion
    // for example, an arbitrarily large regular dodecagon becomes a hexagon
```
guess codespell doesn't catch them all 😉